src/config/config.js

```diff
@@ -67,7 +67,7 @@ function load(app) {
       let defaults = getFullDefaults(app)
       if (defaults) {
         convertOldDefaultsToDeltas(app.config.baseDeltaEditor, defaults)
-        if (app.config.settings.useBaseDeltas) {
+        if (!app.config.settings.keepDefaultsFile) {
```

Comment:
Let's keep backwards compatibility. Use the same setting name, but do the conversion of it's undefined.

Reply:
Agreed. So if true or undefined like now??